Java/other/NKnightKill.java

```diff
@@ -0,0 +1,102 @@
+/**
+It is a backtracking problem in which there are N knights 
+that need to be placed on an N*N board in such a manner so 
+that no two knights can kill each other.
+The code will display all the possible positions of Knights.
+*/
+
+import java.util.Scanner;
+
+public class NKnightKill {
+
+    private static void display(char[][] board) {
+		
+		for (int i=0; i<board.length; i++) {
+			for (int j=0; j<board.length; j++) {
+				System.out.print (board[i][j]+"\t");
+			}
+			System.out.println();
+		}
+	}
+	
+	private static boolean isSafe(char[][] board, int i, int j) {
+		// function will check if is it safe to
+		// place the Knight at i,j
+
+		if (i-2>=0 && j-1>=0 && board[i-2][j-1]=='X') {
+			return false;
+		}
+		if (i-2>=0 && j+1<board.length && board[i-2][j+1]=='X') {
+			return false;
+		}
+		if (i-1>=0 && j-2>=0 && board[i-1][j-2]=='X') {
+			return false;
+		}
+		if (i-1>=0 && j+2<board.length && board[i-1][j+2]=='X') {
+			return false;
+		}
+		return true;
+	}
+	
+	private static void nKnight(char[][] board, int i, int j, int n) {
+
+		if (n==0) {
+			System.out.println("Possibility "+count+" - ");
+			display (board);
+			count++;
+			System.out.println();
+			return;
```

Comment:
Hey @bhav09, you asked me to put a new line at EOF. Please see here, I have already used an Sopln statement to make sure that not only all the boards get printed with a line gap but also for the very last board that will get printed there will definitely be a new line at the end.

Please have a look at this. There is a line gap at the end.
![Untitled](https://user-images.githubusercontent.com/56653257/115066673-7fa76480-9f0d-11eb-9ee8-56031611492d.png)

If I am misunderstanding your point then please elaborate a bit.
Thanks.